packages/suite/src/components/suite/BioAuthGuard/BioAuthGuard.tsx

```diff
@@ -80,36 +61,26 @@ const BioAuthOverlay = ({
                                 alignItems="center"
                                 justifyContent="center"
                             >
-                                {isBioAuthValidationRequired ? (
```

Comment:
So this was here a bit more complex, coz before, the Suite was locked also when it was just blurred await. So that in the background, it wouldn't be visible.

This was scraped just for simplicity with a caveat that it can be added later.

So yeah, it can be simplified, if there is no decision about it.